requirements-extra.txt

```diff
@@ -7,5 +7,6 @@ test_tube>=0.7.5
 wandb>=0.8.21
 trains>=0.14.1
 matplotlib>=3.1.1
+torchvision>=0.4.1
```

Comment:
we have removed it bc it was quite delayed on py3.8 so let's see if they have it there now...

Reply:
You are right, just tested installing 0.4.1 in a py3.8 env and it failed. 
Kinda need it for some doctests. Two options:
1. ignore some of the doctests
2. manually install torchvision only in build docs CI task
what do you think is best?

Reply:
depends on how many tests need it... I would see a bit different option
- ignore the test with `torchvision` if they are just a few
- try to replace `torchvision` by our fund (probably too much work)
- run these tests only on CicleCI as we use there only 3.6/3.7

Reply:
probably the last is the simplest way even we sacrify wider test cases test coverage

Reply:
it seems that we are not testing the doc build in GitHub action anyway now so seems fast the less painful way :]

Reply:
It was actually only 5 to skip. I added a check if torchvision is available and skip the tests. this way we can still tests this code on systems where torchvision is installed.